src/api/roles/client.py

```
@@ -0,0 +1,42 @@
+#models
+from src.api.roles.domain import ClientDetails
+
+
+from src.database.session import get_session
+from src.database.account.models import Account
+from src.database.client.models import Client, FitnessGoals
+
+router = APIRouter(prefix="/roles/client", tags=["client"])
+
+@router.post("/create_client_initial_survey", response_model=CreateClientResponse)
+def create_coach_request(client_details: ClientDetails, db = Depends(get_session), acc: Account = Depends(get_account_from_bearer)):
```

Ruff (0.15.7)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)